src/components/Insight/InsightEditor/InsightEditor.js

```diff
@@ -31,6 +31,10 @@ class InsightEditor extends Component {
     isEditing: false
   }
 
+  trendTag = this.props.tags.find(({ name }) =>
+    name.endsWith('-trending-words')
```

Comment:
@DmitriVanGuard I think, I saw this func somewhere.

Reply:
This function was for filtering